### PR TITLE
CRW-238 set default server and operator...

### PR DESCRIPTION
**Pull request by nickboldt**
CRW-238 set default server and operator image tag to 1.0, not latest; bump to pom to 1.0.2

Change-Id: I607b17856504e802dbf14734ea21577f451f2909
Signed-off-by: nickboldt <nboldt@redhat.com>

### CODE REVIEW

File: operator-installer/config.yaml [hunk 1]
```diff
@@ -20,7 +20,7 @@ data:
 
   # CHE flavor. Upstream `che` or Red Hat `codeready`. Defaults to `che`
   CHE_FLAVOR: "codeready"
-  # Docker image for Che server. Defaults to eclipse/che-server:latest. Keep blank unless you need to deploy your custom image
+  # Docker image for Che server. Defaults to registry.access.redhat.com/codeready-workspaces/server:1.0. Keep blank unless you need to deploy your custom image
```
Comment (slemeur) [inline operator-installer/config.yaml]: "Container image"

**Comment by dmytro-ndp** [comment]
There are another pom.xml's like https://github.com/redhat-developer/codeready-workspaces-deprecated/blob/ba959e0c765ee71782f7372d0e4ddbb22ad76642/pom.xml#L24 which version might be updated as well.

**Comment by nickboldt** [comment]
> There are another pom.xml's like
> 
> [codeready-workspaces-deprecated/pom.xml](https://github.com/redhat-developer/codeready-workspaces-deprecated/blob/ba959e0c765ee71782f7372d0e4ddbb22ad76642/pom.xml#L24)
> 
> Line 24 in [ba959e0](/redhat-developer/codeready-workspaces-deprecated/commit/ba959e0c765ee71782f7372d0e4ddbb22ad76642)
> 
>  <version>1.0.0.GA-SNAPSHOT</version> 
> which version might be updated as well.

No, that's the version of the stacks dependencies, which are unchanged. Only the operator was bumped for 1.0.1 , and now again for 1.0.2.

**Comment by nickboldt** [comment]
merged to 1.0.x

**Pull request closed without merging**